sklearn/preprocessing/label.py

```diff
@@ -304,18 +304,41 @@ def fit(self, y):
         self.classes_ = unique_labels(y)
         return self
 
+    def fit_transform(self, y):
+        """Fit label binarizer and transform multi-class labels to binary
+        labels.
+
+        The output of transform is sometimes referred to by some authors as
+        the 1-of-K coding scheme.
+
+        Parameters
+        ----------
+        y : array or sparse matrix of shape (n_samples,) or \
+(n_samples, n_classes)
```

Comment:
iirc it will actually change the rendering, but I'm still +1 on indenting. We should ask at numpydoc how this is supposed to work ^^